sda/export_test.go

```diff
@@ -48,25 +42,35 @@ func (o *Overlay) TokenToNode(tok *Token) (*TreeNodeInstance, bool) {
 	return tni, ok
 }
 
-func (h *Host) AbortConnections() error {
-	h.closeConnections()
-	close(h.ProcessMessagesQuit)
-	return h.host.Close()
+// AddTree registers the given Tree struct in the underlying overlay.
+// Useful for unit-testing only.
+func (h *Host) AddTree(t *Tree) {
```

Comment:
Having public methods for testing purposes only. I guess there is no way to circumvent this?